bcda/cclf/cclf.go

```diff
@@ -360,14 +335,12 @@ func (importer CclfImporter) ImportCCLFDirectory(filePath string) (success, fail
 		defer c()
 		return importer.FileProcessor.CleanUpCCLF(ctx, cclfMap)
 	}(); err != nil {
-		fmt.Println(err.Error())
-		log.API.Error(err)
+		importer.Logger.Error(err)
 	}
 
 	if failure > 0 {
 		err = errors.New("one or more files failed to import correctly")
```

Comment:
another logging suggestion/non-blocking issue: would it be helpful to include the # of files that failed to import vs a generic message?